lib/services.template.ejs

```diff
@@ -267,7 +267,9 @@ module
       props.forEach(function(name) {
         self[name] = load(name);
       });
-      this.rememberMe = undefined;
+      if ( !this.rememberMe ){
```

Comment:
I think you're right, I was under the impression we needed it because the original code explicitly does that. 
The login function does do 
```
LoopBackAuth.rememberMe = response.config.params.rememberMe !== false;
LoopBackAuth.save();
```
which should provide the `rememberMe` variable with correct value, and I removed it and all tests are still fine, will remove it